### PR TITLE
Add option to trigger tests to CLI

### PR DESCRIPTION
**Pull request by Zlopez**
This is a minimal implementation of #3288.

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: bodhi/client/__init__.py [hunk 1]
```diff
@@ -924,6 +924,36 @@ def waive(update: str, show: bool, test: typing.Iterable[str], comment: str, url
...
 
 
+@updates.command()
+@click.argument('update')
+@openid_option
+@staging_option
+@url_option
+@debug_option
+@handle_errors
+def trigger(update: str, url: str, openid_api: str, **kwargs):
```
Comment (cverna) [inline bodhi/client/__init__.py]: I don't find trigger to be very explicit on what this command does. As a user typing `bodhi update trigger`does not really speek to me.
Maybe something like `bodhi update trigger-tests` or `bodhi update start-tests` would be better. What do you think ?

Comment (Zlopez) [reply bodhi/client/__init__.py]: I wasn't sure how to use option that has two words in it. Looked through bodhi commands and didn't found any. This is why I went for one word option.

File: bodhi/client/__init__.py [hunk 2]
```diff
@@ -924,6 +924,36 @@ def waive(update: str, show: bool, test: typing.Iterable[str], comment: str, url
         print_resp(resp, client)
 
 
+@updates.command()
```
Comment (cverna) [inline bodhi/client/__init__.py]:
```suggestion
@updates.command(name="start-tests")
```

File: bodhi/client/__init__.py (hunk 1 above)
Comment (cverna) [reply bodhi/client/__init__.py]: I think this can be done has suggested above.

File: bodhi/client/__init__.py (hunk 2 above)
Comment (Zlopez) [reply bodhi/client/__init__.py]: Thanks

**Comment by Zlopez** [comment]
Addressed @cverna comments and changed option to `trigger-tests`

**Review by cverna — APPROVED** [review]

**Pull request merged**